static/styles/subscriptions.css

```diff
@@ -986,51 +953,56 @@ h4.stream_setting_subsection_title {
         }
     }
 
-    .subscription_settings {
-        display: none;
-        position: relative;
-        width: 100%;
-        margin: 0 auto;
-        border-radius: 4px;
-        top: -1px;
+    .stream_section {
+        padding-left: 2px;
+        margin: 20px;
```

Comment:
For this change see this comment -> https://github.com/zulip/zulip/pull/21256/files#r815472710